tests/e2e/tests/dashboard/dashboard_test.go

```diff
@@ -41,8 +41,6 @@ import (
 
 const (
 	istioMeshDashboard = "install/kubernetes/helm/istio/charts/grafana/dashboards/istio-mesh-dashboard.json"
-	serviceDashboard   = "install/kubernetes/helm/istio/charts/grafana/dashboards/istio-service-dashboard.json"
-	workloadDashboard  = "install/kubernetes/helm/istio/charts/grafana/dashboards/istio-workload-dashboard.json"
```

Comment:
There are commented out code that refers to the above two constants.
@douglas-reid Can you please comment on this?

Reply:
Yes. That code needs to be revived. It was commented out to unblock a release due to changes in dashboards at the last minute. Those tests are needed and we should not remove these consts.